modules/raw_data_parser/build_tag_push_update.sh

```diff
@@ -0,0 +1,10 @@
+# Run from root repository (NEON-IS-data-processing)
+#!/usr/bin/env bash
+image_name=raw-data-parser
```

Comment:
As we move to the GCP image registry, we'll ensure all image names start with "neon-is". Please migrate to this naming convention at your convenience.

Reply:
A quay.io repo is already created. I will keep this in mind and make the change before we move to the GCP image registry.